2023-03-13 02:08:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table exam (\000d\000a       id bigint generated by default as identity,\000d\000a        [*]day date,\000d\000a        end_time time,\000d\000a        start_time time,\000d\000a        type varchar(255),\000d\000a        year varchar(255),\000d\000a        course_id bigint,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table exam (
       id bigint generated by default as identity,
        day date,
        end_time time,
        start_time time,
        type varchar(255),
        year varchar(255),
        course_id bigint,
        primary key (id)
    ) [42001-210]
2023-03-13 02:08:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "EXAM" not found; SQL statement:

    alter table if exists examlogs 
       add constraint FK89j80youjd0etpt4madsbeewy 
       foreign key (exam_id) 
       reference [42102-210]
2023-03-13 02:11:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table exam (\000d\000a       id bigint generated by default as identity,\000d\000a        [*]day date,\000d\000a        end_time time,\000d\000a        start_time time,\000d\000a        type varchar(255),\000d\000a        year varchar(255),\000d\000a        course_id bigint,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table exam (
       id bigint generated by default as identity,
        day date,
        end_time time,
        start_time time,
        type varchar(255),
        year varchar(255),
        course_id bigint,
        primary key (id)
    ) [42001-210]
2023-03-13 02:11:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "EXAM" not found; SQL statement:

    alter table if exists examlogs 
       add constraint FK89j80youjd0etpt4madsbeewy 
       foreign key (exam_id) 
       reference [42102-210]
2023-03-13 02:25:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table exam (\000d\000a       id bigint generated by default as identity,\000d\000a        [*]day date,\000d\000a        end_time time,\000d\000a        start_time time,\000d\000a        type varchar(255),\000d\000a        year varchar(255),\000d\000a        course_id bigint,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table exam (
       id bigint generated by default as identity,
        day date,
        end_time time,
        start_time time,
        type varchar(255),
        year varchar(255),
        course_id bigint,
        primary key (id)
    ) [42001-210]
2023-03-13 02:25:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "EXAM" not found; SQL statement:

    alter table if exists examlogs 
       add constraint FK89j80youjd0etpt4madsbeewy 
       foreign key (exam_id) 
       reference [42102-210]
2023-03-13 02:27:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table exam (\000d\000a       id bigint generated by default as identity,\000d\000a        [*]day date,\000d\000a        end_time time,\000d\000a        start_time time,\000d\000a        type varchar(255),\000d\000a        year varchar(255),\000d\000a        course_id bigint,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table exam (
       id bigint generated by default as identity,
        day date,
        end_time time,
        start_time time,
        type varchar(255),
        year varchar(255),
        course_id bigint,
        primary key (id)
    ) [42001-210]
2023-03-13 02:27:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "EXAM" not found; SQL statement:

    alter table if exists examlogs 
       add constraint FK89j80youjd0etpt4madsbeewy 
       foreign key (exam_id) 
       reference [42102-210]
2023-03-13 02:36:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table exam (\000d\000a       id bigint generated by default as identity,\000d\000a        end_time time,\000d\000a        exam_day date,\000d\000a        start_time time,\000d\000a        type varchar(255),\000d\000a        [*]year varchar(255),\000d\000a        course_id bigint,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table exam (
       id bigint generated by default as identity,
        end_time time,
        exam_day date,
        start_time time,
        type varchar(255),
        year varchar(255),
        course_id bigint,
        primary key (id)
    ) [42001-210]
2023-03-13 02:36:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "EXAM" not found; SQL statement:

    alter table if exists examlogs 
       add constraint FK89j80youjd0etpt4madsbeewy 
       foreign key (exam_id) 
       reference [42102-210]
2023-03-13 02:40:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table exam (\000d\000a       id bigint generated by default as identity,\000d\000a        end_time time,\000d\000a        exam_day date,\000d\000a        start_time time,\000d\000a        type varchar(255),\000d\000a        [*]year varchar(255),\000d\000a        course_id bigint,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table exam (
       id bigint generated by default as identity,
        end_time time,
        exam_day date,
        start_time time,
        type varchar(255),
        year varchar(255),
        course_id bigint,
        primary key (id)
    ) [42001-210]
2023-03-13 02:40:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "EXAM" not found; SQL statement:

    alter table if exists examlogs 
       add constraint FK89j80youjd0etpt4madsbeewy 
       foreign key (exam_id) 
       reference [42102-210]
2023-03-13 02:49:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table exam (\000d\000a       id bigint generated by default as identity,\000d\000a        end_time time,\000d\000a        exam_day date,\000d\000a        start_time time,\000d\000a        type varchar(255),\000d\000a        [*]year varchar(255),\000d\000a        course_id bigint,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table exam (
       id bigint generated by default as identity,
        end_time time,
        exam_day date,
        start_time time,
        type varchar(255),
        year varchar(255),
        course_id bigint,
        primary key (id)
    ) [42001-210]
2023-03-13 02:49:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "EXAM" not found; SQL statement:

    alter table if exists examlogs 
       add constraint FK89j80youjd0etpt4madsbeewy 
       foreign key (exam_id) 
       reference [42102-210]
2023-03-13 02:52:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table exam (\000d\000a       id bigint generated by default as identity,\000d\000a        end_time time,\000d\000a        exam_day date,\000d\000a        start_time time,\000d\000a        semester varchar(10),\000d\000a        [*]year varchar(4),\000d\000a        course_id bigint,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table exam (
       id bigint generated by default as identity,
        end_time time,
        exam_day date,
        start_time time,
        semester varchar(10),
        year varchar(4),
        course_id bigint,
        primary key (id)
    ) [42001-210]
2023-03-13 02:52:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "EXAM" not found; SQL statement:

    alter table if exists examlogs 
       add constraint FK89j80youjd0etpt4madsbeewy 
       foreign key (exam_id) 
       reference [42102-210]
2023-03-13 03:00:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000d\000a    create table exam (\000d\000a       id bigint generated by default as identity,\000d\000a        end_time time,\000d\000a        exam_day date,\000d\000a        start_time time,\000d\000a        semester varchar(10),\000d\000a        [*]year varchar(4),\000d\000a        course_id bigint,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table exam (
       id bigint generated by default as identity,
        end_time time,
        exam_day date,
        start_time time,
        semester varchar(10),
        year varchar(4),
        course_id bigint,
        primary key (id)
    ) [42001-210]
2023-03-13 03:00:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "EXAM" not found; SQL statement:

    alter table if exists examlogs 
       add constraint FK89j80youjd0etpt4madsbeewy 
       foreign key (exam_id) 
       reference [42102-210]
2023-03-13 03:09:58 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/aomop/OneDrive/Desktop/CaptainCode/ExamSystem/examsystem/data/h2/examsystem.mv.db [2.1.210/7]" [50000-210]
	at org.h2.message.DbException.get(DbException.java:216)
	at org.h2.message.DbException.convert(DbException.java:414)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:120)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3296)
	at org.h2.mvstore.MVStore.panic(MVStore.java:585)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:461)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4056)
	at org.h2.mvstore.db.Store.<init>(Store.java:129)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at com.intellij.database.remote.jdbc.helpers.JdbcHelperImpl.connect(JdbcHelperImpl.java:653)
	at com.intellij.database.remote.jdbc.impl.RemoteDriverImpl.connect(RemoteDriverImpl.java:63)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/aomop/OneDrive/Desktop/CaptainCode/ExamSystem/examsystem/data/h2/examsystem.mv.db [2.1.210/7]" [50000-210]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:573)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	... 34 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/aomop/OneDrive/Desktop/CaptainCode/ExamSystem/examsystem/data/h2/examsystem.mv.db [2.1.210/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:444)
	... 28 more
2023-03-13 03:10:49 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-03-13 03:10:52 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-03-13 03:11:57 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/aomop/OneDrive/Desktop/CaptainCode/ExamSystem/examsystem/data/h2/examsystem.mv.db [2.1.210/7]" [50000-210]
	at org.h2.message.DbException.get(DbException.java:216)
	at org.h2.message.DbException.convert(DbException.java:414)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:120)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3296)
	at org.h2.mvstore.MVStore.panic(MVStore.java:585)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:461)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4056)
	at org.h2.mvstore.db.Store.<init>(Store.java:129)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at com.intellij.database.remote.jdbc.helpers.JdbcHelperImpl.connect(JdbcHelperImpl.java:653)
	at com.intellij.database.remote.jdbc.impl.RemoteDriverImpl.connect(RemoteDriverImpl.java:63)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/aomop/OneDrive/Desktop/CaptainCode/ExamSystem/examsystem/data/h2/examsystem.mv.db [2.1.210/7]" [50000-210]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:573)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	... 34 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/aomop/OneDrive/Desktop/CaptainCode/ExamSystem/examsystem/data/h2/examsystem.mv.db [2.1.210/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:444)
	... 28 more
2023-03-13 03:14:28 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-03-13 03:14:46 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:470)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
